imports/plugins/included/discount-codes/server/methods/methods.app-test.js

```diff
@@ -17,7 +17,6 @@ const code = {
 
 before(function () {
   this.timeout(10000);
-  Meteor._sleepForMs(7000);
```

Comment:
We use this sleep to make sure fixtures are completely loaded before running tests on older systems